__tests__/hooks/useStartDictate.js

```diff
@@ -0,0 +1,22 @@
+import { timeouts } from '../constants.json';
+
+import isDictating from '../setup/pageObjects/isDictating';
+import uiConnected from '../setup/conditions/uiConnected';
+
+// selenium-webdriver API doc:
+// https://seleniumhq.github.io/selenium/docs/api/javascript/module/selenium-webdriver/index_exports_WebDriver.html
+
+jest.setTimeout(timeouts.test);
+
+test('calling startDictate should start dictate', async () => {
+  const { driver, pageObjects } = await setupWebDriver({
+    props: {
+      webSpeechPonyfillFactory: () => window.WebSpeechMock
+    }
+  });
+
+  await driver.wait(uiConnected(), timeouts.directLine);
+  await pageObjects.runHook('useStartDictate', [], startDictate => startDictate());
+
+  await expect(pageObjects.isDictating()).resolves.toBeFalsy();
```

Comment:
Why is `pageObjects.isDictating()` falsy after we start dictating... Seems counterintuitive

Reply:
is this resolved?

Reply:
I added comments. This is intended.

The `startDictate` function will start the dictation engine, but doesn't necessarily mean the engine is dictating.